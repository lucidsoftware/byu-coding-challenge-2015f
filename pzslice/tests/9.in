1 987
81 51
19 17
52 92
38 31
10 8
24 62
46 55
44 37
78 60
27 90
24 71
41 27
37 99
93 4
33 49
99 11
14 16
44 84
100 8
48 81
41 95
2 12
69 83
6 83
19 21
78 28
94 18
71 43
66 56
4 97
30 45
38 9
9 87
93 17
17 34
20 73
35 38
48 81
4 74
37 1
13 42
95 69
85 24
52 2
58 28
10 75
78 10
56 84
86 19
16 44
37 95
73 45
10 16
87 97
63 44
73 17
37 90
77 50
13 63
56 36
94 72
3 41
87 26
93 74
63 2
31 35
35 4
79 6
38 37
35 42
42 10
36 52
2 45
23 98
77 41
63 29
77 25
40 46
60 54
74 15
23 67
19 61
19 25
91 52
4 10
13 81
10 7
64 84
96 8
87 11
51 35
43 62
22 84
21 33
72 20
67 5
67 39
22 24
93 95
55 53
21 36
89 28
20 7
71 76
32 11
58 45
96 19
12 32
10 43
28 69
32 62
91 30
14 79
75 36
98 96
11 45
51 84
92 49
44 58
91 80
32 24
23 98
38 48
62 34
22 34
33 5
70 41
58 74
59 15
37 92
91 69
55 72
73 87
49 9
80 75
18 66
35 97
74 4
62 52
93 60
59 74
71 29
69 59
9 98
64 85
26 33
55 39
46 39
79 56
68 3
36 15
15 32
67 46
82 72
48 94
51 36
94 6
3 97
16 92
7 26
64 14
47 69
98 35
96 5
87 63
92 85
8 53
67 5
66 53
72 40
90 73
7 97
51 1
4 47
7 48
13 22
15 77
46 55
14 1
16 45
37 54
16 43
83 80
32 13
91 57
94 85
78 43
59 82
20 30
30 75
18 44
91 40
59 96
48 19
75 98
19 34
78 39
57 41
95 35
81 99
2 84
79 36
53 69
90 8
49 87
55 45
69 96
3 33
70 45
27 80
98 98
47 54
65 79
53 39
91 3
45 49
6 35
97 65
22 85
59 7
91 61
75 22
18 21
99 80
24 16
95 45
2 88
62 79
58 6
88 61
96 9
55 79
59 85
74 79
94 27
86 23
78 45
39 24
15 38
55 91
47 31
58 79
41 42
57 87
79 32
59 23
39 68
18 99
87 82
33 33
41 40
25 64
53 21
90 32
6 83
87 82
33 26
43 22
28 94
77 67
77 42
37 66
81 64
92 65
86 78
25 79
75 52
84 38
25 10
76 8
94 6
95 87
26 31
33 70
93 47
2 25
93 57
50 8
19 83
74 79
72 79
58 66
14 18
38 48
67 98
62 21
84 50
52 78
46 59
98 85
76 33
96 90
16 53
56 23
31 48
97 22
95 83
49 71
72 87
48 21
61 34
10 85
52 12
3 7
7 77
66 44
10 86
79 3
57 18
56 44
94 69
93 83
6 68
83 67
78 61
44 55
63 7
15 93
39 100
95 74
40 73
62 83
82 79
72 43
28 59
17 74
60 57
93 36
97 94
29 31
13 45
71 65
23 91
54 42
73 29
12 44
21 80
76 16
21 71
82 66
37 77
8 48
90 48
79 44
20 70
77 10
81 90
93 42
49 38
80 61
49 81
37 9
41 41
64 98
62 49
11 44
67 49
52 90
35 97
17 96
47 4
85 85
28 59
18 62
45 39
17 87
15 1
70 70
72 11
82 29
44 1
38 31
97 47
20 23
78 77
13 49
96 2
46 49
94 50
21 38
46 77
96 31
38 85
5 52
15 99
64 24
89 82
73 48
74 42
23 41
23 15
12 24
64 99
6 83
2 15
67 69
81 38
50 9
49 30
20 23
93 14
58 92
82 91
73 11
13 31
97 63
36 86
26 86
23 82
80 46
12 44
74 32
17 58
17 14
45 26
85 17
77 52
87 92
57 100
86 4
91 34
55 9
77 37
80 55
24 22
32 78
71 14
7 44
3 75
33 87
12 98
29 79
42 38
54 91
20 72
70 2
82 60
50 75
33 93
43 79
51 79
38 5
69 84
56 57
5 31
99 60
62 15
54 81
87 2
63 77
70 88
89 7
93 33
69 91
84 21
63 29
78 23
1 61
72 83
29 76
79 29
45 95
54 21
55 84
46 67
61 24
82 9
19 36
12 95
36 80
83 80
53 21
56 40
16 57
63 75
91 98
46 9
93 82
38 47
31 27
3 50
37 27
28 86
45 45
64 65
26 98
34 65
95 18
49 95
1 66
73 84
96 87
29 80
48 96
23 52
66 5
82 63
58 91
24 20
98 86
54 5
10 85
42 78
23 79
56 21
32 42
13 36
31 96
26 74
91 7
62 49
44 82
27 90
65 64
42 77
24 1
61 29
93 98
73 84
70 15
14 88
95 7
94 28
2 88
59 70
92 87
90 14
54 8
46 26
62 68
43 97
14 36
86 15
8 5
15 6
48 89
33 75
29 38
38 59
19 85
92 44
57 43
87 5
92 83
10 39
67 91
20 38
1 9
61 97
45 95
8 55
81 30
90 22
66 56
50 81
63 91
13 32
58 40
2 1
9 96
86 31
88 39
86 54
26 32
100 78
50 69
12 7
91 3
87 62
33 24
73 9
50 58
20 38
70 20
31 32
14 82
27 93
6 52
41 39
49 13
44 89
58 74
58 60
72 63
46 52
20 18
68 82
48 29
75 3
92 44
89 89
39 65
47 31
34 23
43 98
57 82
3 93
58 10
32 26
96 68
17 2
18 8
67 37
60 41
1 15
60 42
47 14
13 77
23 69
27 4
5 93
21 75
66 98
18 74
49 79
91 20
76 2
11 80
94 65
17 100
94 64
71 13
1 39
26 91
71 79
100 93
83 17
81 99
84 42
85 48
10 36
30 94
68 76
97 79
59 3
9 76
82 2
60 49
62 42
73 96
91 79
62 33
68 33
9 38
25 40
28 28
47 28
13 21
15 48
31 84
2 34
37 64
74 37
61 76
61 72
12 54
56 27
14 47
88 83
43 59
99 97
20 100
19 40
30 2
17 38
64 59
74 28
49 65
83 89
8 75
25 71
49 79
65 55
39 52
51 47
69 35
57 42
74 34
66 100
71 87
54 50
97 78
6 98
17 37
73 25
16 91
42 7
86 58
78 30
16 20
26 88
2 62
55 41
72 87
95 2
55 49
78 42
94 26
5 57
10 49
53 73
32 79
84 1
3 1
69 38
39 66
30 12
60 93
14 26
62 76
42 24
14 13
55 100
12 95
56 35
31 69
90 3
33 87
65 4
51 81
56 87
91 56
90 92
9 59
69 39
57 60
99 77
11 99
56 32
78 22
4 82
62 51
51 43
33 4
13 31
62 90
8 39
95 36
42 71
52 26
22 45
8 70
52 79
9 21
96 47
35 28
38 25
60 23
48 60
4 9
17 95
12 33
50 58
74 85
22 75
33 9
40 30
45 35
12 14
97 92
24 85
52 74
35 87
84 65
19 68
82 23
13 41
53 84
82 3
3 53
86 48
22 53
14 96
85 76
67 98
52 43
27 99
45 42
89 70
38 16
59 5
7 82
69 39
91 88
86 35
54 53
48 37
3 14
93 80
77 33
79 73
51 11
84 43
67 50
60 78
74 10
32 40
62 67
61 53
7 10
87 41
20 23
23 74
15 76
80 33
47 67
21 39
85 51
72 37
47 10
22 59
33 11
88 50
72 34
24 69
43 70
59 89
38 24
71 65
25 70
51 46
54 59
48 18
37 86
70 77
96 11
13 34
44 18
63 99
71 83
64 5
99 80
56 81
44 37
97 94
44 72
26 42
93 20
21 40
51 55
29 95
3 49
82 36
59 51
95 27
80 42
85 21
81 14
20 9
26 8
99 95
86 77
50 24
25 75
79 50
44 44
28 34
78 52
9 96
23 21
99 50
71 57
29 43
30 90
60 84
40 87
17 30
63 29
91 37
26 36
24 6
3 21
88 27
22 24
38 9
12 75
39 96
80 35
61 76
87 97
86 12
46 94
80 6
2 3
14 71
66 5
74 28
64 32
94 87
33 11
59 78
52 89
29 40
83 52
95 78
74 41
68 74
65 70
83 90
10 52
60 36
30 83
41 77
34 96
31 53
25 49
85 95
10 25
45 18
27 14
77 27
92 93
64 93
43 77
65 72
44 82
45 8
76 83
18 40
89 89
20 92
90 93
43 69
4 76
42 50
77 73
70 39
24 16
59 64
40 7
41 53
100 25
31 4
15 40
79 54
49 55
4 100
77 46
12 57
94 81
87 38
1 49
19 7
20 6
45 10
59 46
4 7
69 72
36 28
69 61
21 12
98 18
15 48
75 91
58 79
80 84
91 9
82 39
93 79
59 84
68 69
68 32
97 87
97 80
68 24
17 94
9 83
17 98
41 21
20 8
79 42
24 7
52 19
14 48
88 87
51 26
8 28
66 76
95 93
75 87
63 1
66 58
60 6
39 92
17 5
26 87
97 66