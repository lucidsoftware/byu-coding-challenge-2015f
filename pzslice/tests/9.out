9603.99942756
